### PR TITLE
Add button to return from Character Selection Scene.

### PR DESCRIPTION
**Pull request by levyadams**
Button added so players can return to the main menu.

### CODE REVIEW

File: game/src/js/scenes/CharacterSelectionScene.js [hunk 1]
```diff
@@ -60,6 +60,17 @@ export default class CharacterSelectionScene extends Phaser.Scene {
     chkButton.on('pointerover', () => {
       this.showCharacter(btnData.key);
     });
+    
+    let backToMenuButton = new Button(this, 450, 610, 'Return To Menu!');
```
Comment (ignacioxd) [inline game/src/js/scenes/CharacterSelectionScene.js]: This is mostly stylistic, but I'd put just "BACK".

File: game/src/js/scenes/CharacterSelectionScene.js [hunk 2]
```diff
@@ -60,6 +60,17 @@ export default class CharacterSelectionScene extends Phaser.Scene {
...
+    backToMenuButton.props={
```
Comment (ignacioxd) [inline game/src/js/scenes/CharacterSelectionScene.js]: These custom settings should be passed into the constructor. Right now this is just overwriting the props with a subset of the necessary variables, which is likely to cause problems.

File: game/src/js/scenes/CharacterSelectionScene.js [hunk 3]
```diff
@@ -60,6 +60,17 @@ export default class CharacterSelectionScene extends Phaser.Scene {
...
+    fontColorNormal: '#0000ff',
+    fontColorOver: '#458b74',
+    backgroundColorOver: '#ffffff55'
+    }
+    backToMenuButton.setFill(backToMenuButton.props.fontColorNormal);
```
Comment (ignacioxd) [inline game/src/js/scenes/CharacterSelectionScene.js]: This line will not be necessary once custom settings are passed in properly.

**Comment by levyadams** [comment]
Added all the things!

File: game/src/js/scenes/CharacterSelectionScene.js [hunk 4]
```diff
@@ -61,13 +61,12 @@ export default class CharacterSelectionScene extends Phaser.Scene {
...
     });
     
-    let backToMenuButton = new Button(this, 450, 610, 'Return To Menu!');
-    backToMenuButton.props={
-    fontColorNormal: '#0000ff',
-    fontColorOver: '#458b74',
-    backgroundColorOver: '#ffffff55'
-    }
-    backToMenuButton.setFill(backToMenuButton.props.fontColorNormal);
+    let backToMenuButton = new Button(this, 450, 610, 'Return To Menu!',props={
```
Comment (ignacioxd) [inline game/src/js/scenes/CharacterSelectionScene.js]: `props=` is not needed:

```javascript
let backToMenuButton = new Button(this, 450, 610, 'BACK', {
      backgroundColorOver: '#ffffff55',
      fontColorNormal: '#0000ff',
      setFill:'#0000ff',
      fontColorOver: '#458b74',
    });
```

**Pull request merged**